File: united-airlines-training/Models/DistanceLearningMedia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace united_airlines_training.Models
{
   public class DistanceLearningMedia
   {
      public int MediaId { get; set; }

      public string Instructions { get; set; }

      public List<DistanceLearningSession> Sessions { get; set; }

      public List<DistanceLearningAttendanceRequirement> AttendanceRequirements { get; set; }


   }
}

File: united-airlines-training/ClientApp/projects/app/src/app/models/distance-learning-media.ts
import { DistanceLearningSession } from './distance-learning-session';
import { DistanceLearningAttendanceRequirement } from './DistanceLearningAttendanceRequirement';

export interface DistanceLearningMedia {
  mediaId: number,
  instructions: string,
  sessions?: DistanceLearningSession[],
  attendanceRequirements?: DistanceLearningAttendanceRequirement[]
}
